docs/usage.rst

```diff
@@ -5,3 +5,47 @@ Usage
 To use TIA Toolbox in a project::
 
     import tiatoolbox
+
+
```

Comment:
fine- but should we not ensure that changes are specific to the mentioned PR title?

Reply:
Need to add help as well for the feature.